README.md

```diff
@@ -69,6 +69,12 @@ Instanciate client object and set default configurations.
     * default schema name
   * checkInterval [integer]
     * interval milliseconds of each RPC to check query status (default: 800ms)
+  * enable_verbose_state_callback [boolean]
```

Comment:
My comment did a wrong guidance :(
It should be `enableVerboseStateCallback` because all other parameters are named in camelCase.
I'll fix it after merging.